src/renderers/shared/fiber/isomorphic/ReactPortal.js

```diff
@@ -27,6 +27,7 @@ exports.createPortal = function(
   implementation: any,
   key: ?string = null,
 ): ReactPortal {
+  containerInfo._reactPortalContainer = {};
```

Comment:
>Absolutely we should check for the presence of '_reactPortalContainer' before adding another one. Will fix.

It seems like if we change this to be a boolean then it’s not a problem and we can always set it. Does it have to be an object?

>I could rename '_reactPortalContainer' to 'unstable_reactPortalContainer' to additionally show that it's not to be relied on.

I think we use `unstable_` for actual APIs people *might* call. In this case something like `__reactInternalIsPortalContainer` would look better IMO.

>So if somehow createPortal was called after the portal component unmounted that would be a problem.

I don’t think this is possible. So seems fine.